### PR TITLE
ft: setup register endpoint for sellers

### PR DESCRIPTION
**Pull request by tolls-3**
## Description
Add .env example file and validation packages
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by tolls-3** [comment]
[API endpoint for sellers Registration](https://trello.com/c/UpPMzzwQ/10-api-endpoint-for-sellers-registration)

**Comment by Rexben001** [comment]
@tolls-3 Is it ready for review?
Can you add nyc and coveralls packages?

**Review by Rexben001 — CHANGES_REQUESTED** [review]
Replace `  mongoURI = process.env.DB_CONNECTION_TEST
` with `mongoURL = process.env.DB_CONNECTION_TEST || 'mongodb://localhost/test'`. You were not meant to remove it because of TravisCI config

**Review by Rexben001 — APPROVED** [review]

File: tests/server.test.js [hunk 1]
```diff
@@ -2,10 +2,9 @@ const request = require('supertest')
 const server = require('../server')
 
 describe('index route', () => {
-  test('it runs', async done => {
+  it('it runs', async () => {
```
Comment (Rexben001) [inline tests/server.test.js]: For your async events, add `done`. 
Eg. it('it runs', async done => {
    const res = await request(server).get('/')
    expect(res.text).toBe('Api is running!!')
    expect(res.status).toBe(200)
    done()
  })

**Review by Rexben001 — CHANGES_REQUESTED** [review]

File: tests/server.test.js (hunk 1 above)
Comment (tolls-3) [reply tests/server.test.js]: ok

**Review by Rexben001 — APPROVED** [review]

**Comment by Rexben001** [comment]
**Feedbacks**
- Rename `password2` to `confirm_password`
- In your server.js, you are not catering for invalid URL
- If I don't pass in `password2`, I won't get an error msg requiring me to supply `password`

Overall, good job guys! 🔥 🔥

File: server.js [hunk 2]
```diff
@@ -3,18 +3,30 @@ const express = require('express')
...
+    console.log('MongoDB is connected')
+  })
+
+  .catch(err => console.log(err))
 
 server.get('/', (req, res) => {
   res.status(200).send('Api is running!!')
 })
 
+server.get('*', (req, res) => {
```
Comment (Rexben001) [inline server.js]: You only add it for `.get` method. Replace `.get` with `.all`

**Pull request merged**